examples/federated_eval_example/client.py

```diff
@@ -44,13 +44,7 @@ def get_criterion(self, config: Config) -> _Loss:
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="FL Client Main")
     parser.add_argument("--dataset_path", action="store", type=str, help="Path to the local dataset")
-    parser.add_argument(
-        "--checkpoint_path",
-        action="store",
-        type=str,
-        help="Path to client model checkpoint.",
-        required=False,
-    )
+    parser.add_argument("--checkpoint_path", action="store", type=str, help="Path to client model checkpoint.")
```

Comment:
This is not really optional, without the checkpoint the client fails, so I removed the `required=False` from it.

Reply:
So this should work gracefully, as long as either the server or the client provides a model checkpoint path. That is, at least one of them **needs** to be defined, but both do not. I think the reason you're seeing a failure on the client is that both the client and the server are not providing a path. 

I'm not sure what the right thing to do is in the sense that it is _technically_ optional but conditioned on the presence of a server checkpoint. My inclination is too keep this option, as otherwise it signals to the user that they must have a client side checkpoint. Very minor though.